syntax="proto2";

package proto;

message SupplierP {
  required int32 supplierKey = 1;
  required string name = 2;
  required string address = 3;
  required int32 nationKey = 4;
  required string phone = 5;
  required double accbal = 6;
  required string comment = 7;
}
  
 